Rsp.NotifyFunction/Services/RSPNotifyService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Extensions.Logging;
+using Notify.Client;
+using Notify.Models.Responses;
+using Rsp.Logging.Extensions;
+using Rsp.NotifyFunction.Contracts;
+using Rsp.NotifyFunction.Models;
+
+namespace Rsp.NotifyFunction.Services
+{
+    public class RSPNotifyService(IRSPNotifyClient notifyClient, ILogger<RSPNotifyService> logger) : IRSPNotifyService
+    {
+        private readonly NotificationClient govUkNotifyClient = notifyClient.GetClient();
+
+        public bool SendEmail(EmailNotificationMessage emailNotificationMessage)
+        {
+            var result = false;
+            try
+            {
+                EmailNotificationResponse response = govUkNotifyClient
```

Comment:
we use the `var` keyword to define the variables, as the type will be inferred from right hand side. Unless, you can't use `var`. Just like you did on line 16. Make it consistent everywhere please.

```suggestion
                var response = govUkNotifyClient
```